notebooks/Generalized Substructure Search.ipynb

```diff
@@ -75,20 +75,20 @@
     "from rdkit.Chem import rdSubstructLibrary\n",
     "import pickle, time\n",
     "import gzip\n",
+    "import chembl_downloader\n",
     "\n",
-    "gz = gzip.GzipFile('/home/glandrum/Downloads/chembl_29.sdf.gz')\n",
-    "suppl = Chem.ForwardSDMolSupplier(gz)\n",
     "RDLogger.DisableLog(\"rdApp.warning\")\n",
-    "t1=time.time()\n",
-    "data = []\n",
-    "for i,mol in enumerate(suppl):\n",
-    "    if not ((i+1)%50000):\n",
-    "        print(f\"Processed {i+1} molecules in {(time.time()-t1):.1f} seconds\")\n",
-    "    if mol is None or mol.GetNumAtoms()>50:\n",
-    "        continue\n",
-    "    fp = Chem.PatternFingerprint(mol,fpSize=1024,tautomerFingerprints=True)\n",
-    "    smi = Chem.MolToSmiles(mol)\n",
-    "    data.append((smi,fp))\n",
+    "with chembl_downloader.supplier(version=\"29\") as suppl:\n",
```

Comment:
I'd suggest modifying adding a comment above this mentioning that it's using your chembl_downloader and linking to the github repo and possibly your blog post.

Reply:
Sorry for taking so long, but I have added the requested additional text in 09befa2. Cheers!